io.fairyproject.platforms/core-platform/src/main/java/io/fairyproject/module/ModuleDownloader.java

```diff
@@ -82,6 +87,21 @@ private File locateLocalRepo() {
     public static Path download(String groupId, String artifactId, String version, @Nullable String customRepository) throws IOException {
         Files.createDirectories(MODULE_DIR);
 
+        if (Debug.IN_FAIRY_IDE) {
```

Comment:
We are already locating local repo in contructor, I don't think there is any reason to do this change?